pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -544,10 +507,10 @@ func TestEtcdCreateBinding(t *testing.T) {
 		},
 	}
 	for k, test := range testCases {
-		storage, bindingStorage, _, fakeClient := newStorage(t)
+		storage, bindingStorage, _, server := newStorage(t)
```

Comment:
I added the explicit terminate at the end similar to the other example. 

Same comment on cleanup I think would apply here.